pages/[resourceType]/[id]/evaluate.tsx

```diff
@@ -54,9 +54,9 @@ const EvaluateMeasurePage = () => {
   const [loadingRequest, setLoadingRequest] = useState(false);
   const [measureReportBody, setMeasureReportBody] = useState("");
   const [gridColSpans, setGridColSpans] = useState([3, 3, 0]);
-
   const [practitionerValue, setPractitionerValue] = useState("");
   const [patientValue, setPatientValue] = useState("");
+  const [groupValue, setGroupValue] = useState("");
```

Comment:
Now that we have group functionality, we can add a quick sentence to the `EvaluatePage()` JSDoc (it's a few lines up) about the Group component. Could be something like "The Group SelectComponent only appears if the reportType selected is "Population""